docs/pages/serving_a_cache.mdx

```diff
@@ -92,16 +66,19 @@ This section documents the additional arguments you can pass to the command abov
 
 There are other configurations available to modify via the configuration file. Refer to the [Parameters page](./parameters.mdx) for details.
 
-## Test CacheFunctionality
+## Test Cache Functionality
 
-Once you have your cache set up, follow the steps below to test if your cache can access a file through a Pelican federation. 
+Once you have your cache set up, follow the steps below to test if your cache can access a file through a Pelican federation.
 
-1. Have data available that is accessible via an origin. Since a cache only works with data available via an origin that’s in the same data federation, the following assumes that you have a an accessible namespace within the same data federation. For more information on how to set that up, refer to [Serving an Origin](./serving_an_origin.mdx). Ensure there is some public file available.
-2. Curl the director for the test file and ensure that the url used is the cache that you set up. Assuming your directory is `/tmp/demo`, run the following command to curl your known public file from the cache. 
-``` $ curl -v https://<cache-url>/<namespace/path-to-public-file> ```  Where: 
+1. Have data available and accessible via an origin. Since a cache only works with data available via an origin that’s in the same Pelican federation, the following assumes that you have a an accessible namespace within the same Pelican federation. For more information on how to set that up, refer to [Serving an Origin](./serving_an_origin.mdx). The following also assume that the object to test is public file. To set up a public file through a Pelican origin, refer to the [Configuration documentation](./parameters.mdx#Origin-Exports).
+2. Curl the director for the test file and ensure that the url used is the cache that you set up. Assuming your directory is `/tmp/demo`, run the following command to curl your known public file from the cache.
+
+```$ curl -v https://<cache-url>/<namespace/path-to-public-file>```
```

Comment:
There should be a new line between ``` and the command (and after the command) to make this a code block instead of inline. My markdown renderer still render this into a inline code.
It's like this (desired):
```
$ curl
```

VS this (current):
```$ curl ```